src/transformers/models/roberta/configuration_roberta.py

```diff
@@ -59,6 +59,12 @@ class RobertaConfig(BertConfig):
     """
     model_type = "roberta"
 
-    def __init__(self, pad_token_id=1, bos_token_id=0, eos_token_id=2, **kwargs):
+    def __init__(self, pad_token_id=1, bos_token_id=0, eos_token_id=2, normalize_embeddings=True, **kwargs):
         """Constructs RobertaConfig."""
-        super().__init__(pad_token_id=pad_token_id, bos_token_id=bos_token_id, eos_token_id=eos_token_id, **kwargs)
+        super().__init__(
+            pad_token_id=pad_token_id,
+            bos_token_id=bos_token_id,
+            eos_token_id=eos_token_id,
+            normalize_embeddings=normalize_embeddings,
```

Comment:
Correct way :thinking:

Reply:
we should not put in `super().__init__(...)` since it'll apply to BERT's config then as well. It should only be for RoBERTa